### PR TITLE
new module vcflib/vcfwave

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/vcflib/vcfwave/meta.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: "vcflib_vcfwave"
+description: write your description here
```
Comment (matthdsm) [inline modules/nf-core/vcflib/vcfwave/meta.yml:3]: Please do 😛

Comment (nvnieuwk) [reply modules/nf-core/vcflib/vcfwave/meta.yml:3]: Gotta get this stupid module working first :grin:

**Comment by SPPearce** [comment]
Unusual that singularity and docker are giving different md5sums for the vcf, normally that's conda doing that. Be worth checking what the actual diff is.

Why does conda fail with the tool? Can that be fixed in bioconda? 
I think there is some way that we've been incorporating a check for conda and a clear error message into the main.nf, but on my phone ATM so searching for that is hard.

**Comment by SPPearce** [comment]
My suspicion is that there is a header line difference between the two containers, as happens in conda for vcf files.

**Comment by nvnieuwk** [comment]
Yeah I'm not sure what exactly is the problem, I found another tool that decomposes better when creating this modules (hence why I stopped working on it). I'll pick it back up later, but it's not my priority anymore :grin:

**Pull request closed without merging**